webview/platforms/edgechromium.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+
+"""
+(C) 2014-2019 Roman Sirokov and contributors
+Licensed under BSD license
+
+http://github.com/r0x0r/pywebview/
+"""
+
+import os
+import sys
+import logging
+import json
+import shutil
+import tempfile
+import webbrowser
+from threading import Event, Semaphore
+from ctypes import windll
+from platform import architecture
+
+from webview import WebViewException, _debug, _user_agent
+from webview.serving import resolve_url
+from webview.util import parse_api_js, interop_dll_path, parse_file_type, inject_base_uri, default_html, js_bridge_call
+from webview.js import alert
+from webview.js.css import disable_text_select
+
+import clr
+
+
+clr.AddReference('System.Windows.Forms')
+clr.AddReference('System.Collections')
+clr.AddReference('System.Threading')
+
+import System.Windows.Forms as WinForms
+from System import IntPtr, Int32, String, Action, Func, Type, Environment, Uri
+from System.Threading.Tasks import Task, TaskScheduler, TaskContinuationOptions
+from System.Drawing import Size, Point, Icon, Color, ColorTranslator, SizeF
+
+archpath = 'x64' if architecture()[0] == '64bit' else 'x86'
+os.environ['Path'] = interop_dll_path(archpath) + ';' + os.environ['Path']
+clr.AddReference(interop_dll_path('Microsoft.Web.WebView2.Core.dll'))
+clr.AddReference(interop_dll_path('Microsoft.Web.WebView2.WinForms.dll'))
+from Microsoft.Web.WebView2.WinForms import WebView2, CoreWebView2CreationProperties
+from Microsoft.Web.WebView2.Core import CoreWebView2Environment
+
+logger = logging.getLogger('pywebview')
+
+class EdgeChrome:
+    def __init__(self, form, window):
+        self.pywebview_window = window
+        self.web_view = WebView2()
+        props = CoreWebView2CreationProperties()
+        #props.UserDataFolder = os.path.join(os.getcwd(), 'profile')
+        props.UserDataFolder = os.path.join(os.environ['LOCALAPPDATA'], 'pywebview')
+        self.web_view.CreationProperties = props
+        form.Controls.Add(self.web_view)
+
+        self.js_result_semaphore = Semaphore(0)
+        self.web_view.Dock = WinForms.DockStyle.Fill
+        #settings under on_webview_ready 
+        self.web_view.CoreWebView2Ready += self.on_webview_ready
+        self.web_view.NavigationStarting += self.on_navigation_start
+        self.web_view.NavigationCompleted += self.on_navigation_completed
+        self.web_view.WebMessageReceived += self.on_script_notify
+
+        self.httpd = None # HTTP server for load_html
+        self.tmpdir = None
+        self.url = None
+        self.ishtml = False
```

Comment:
All the http server related stuff can be purged

Reply:
The http server related stuff has been purged,